apps/webpack.js

```diff
@@ -16,18 +16,20 @@ var baseConfig = {
       repl: path.resolve(__dirname, 'src/noop'),
     }
   },
-  externals: {
-    "johnny-five": "var JohnnyFive",
-    "playground-io": "var PlaygroundIO",
-    "chrome-serialport": "var ChromeSerialport",
-    "marked": "var marked",
-    "blockly": "this Blockly",
-    "react": "var React",
-    "react-dom": "var ReactDOM",
-    "jquery": "var $",
-    "radium": "var Radium",
-    "bindings": true
-  },
+  externals: [
+    {
+      "johnny-five": "var JohnnyFive",
+      "playground-io": "var PlaygroundIO",
+      "chrome-serialport": "var ChromeSerialport",
+      "marked": "var marked",
+      "blockly": "this Blockly",
+      "react": "var React",
+      "react-dom": "var ReactDOM",
+      "jquery": "var $",
+      "radium": "var Radium",
+      "bindings": true
+    }
+  ],
```

Comment:
I'm actually just going to remove this since it gets overridden every time anyhow.